backend/app/routers/custom_lists.py

```diff
@@ -45,7 +45,7 @@ async def delete_custom_list(id: uuid.UUID, auth: GoogleAuth = Depends(decode_jw
     return await update_user_custom_lists_remove(db_client, email=auth.email, id=id)
 
 
-@router.post("{streamchaser_id}", response_model=UpdateCustomListAddResult | None)
+@router.post("/{list_id}", response_model=UpdateCustomListAddResult | None)
```

Comment:
We changed what the query returns

```suggestion
@router.post("/{list_id}", response_model=list[UpdateUserCustomListsAddResult])
```

Reply:
Done

Reply:
This actually broke the backend, and i reverted back to what it previously was

Reply:
Weird, it should be what the function returns